index.html

```diff
@@ -201,25 +201,43 @@ <h2>Music</h2>
     <section id="about">
         <div class="card">
             <h2>What is this?</h2>
-            <p>The demoscene is a computer art subculture focused on producing demos.</p>
-            
             <p>
-                Demos are digital art creations at the crossroad of short films, music videos and video games. Although they present
-                a non-interactive experience, often music driven, like a music video does, they are rendered in real-time like video
-                games are. The purpose of a demo is to show off programming, visual art, and musical skills. Demos and other demoscene productions (graphics, music, videos, games)
-                are shared at festivals known as demoparties, voted on by those who attend and released online.
+                64k scene is a website about so called <em>64k intros</em>, a kind of size constrained digital art from the demoscene.
+            </p>
+
+            <blockquote>
+                <p>
+                    One of the core competencies of the demoscene is size-coding, which tries to make the program code as lean as possible. In addition, the tracking of music as a programming-like composing technique and the hacking attitude aimed at mastering new hardware platforms play important roles in the scene. In all those instances, self-limitation is an important creative driver.
```

Comment:
maybe explain what the demoscene / the demos are, before using this quote?

Reply:
Moved the quote right before the paragraph explaining the goal of the website.